packages/flutter/lib/src/material/text_field.dart

```diff
@@ -766,6 +768,17 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@macro flutter.widgets.EditableText.spellCheckConfiguration}
```

Comment:
It's probably also useful to add a paragraph about the default behavior, especially since it is different than in CupertinoTextField and EditableText.